integration/testutil.go

```diff
@@ -77,13 +81,14 @@ const (
 
 // These are images that we use in our integration tests
 const (
-	alpineImage   = "alpine:3.17.1"
-	nginxImage    = "nginx:1.23.3"
-	ubuntuImage   = "ubuntu:23.04"
-	drupalImage   = "drupal:10.0.2"
-	rabbitmqImage = "rabbitmq:3.11.7"
+	alpineImage       = "alpine:3.17.1"
+	nginxImage        = "nginx:1.23.3"
+	ubuntuImage       = "ubuntu:23.04"
+	drupalImage       = "drupal:10.0.2"
+	rabbitmqImage     = "rabbitmq:3.11.7"
+	rabbitmqShaImage  = "rabbitmq@sha256:603be6b7fd5f1d8c6eab8e7a234ed30d664b9356ec1b87833f3a46bb6725458e"
+	rethinkdbShaImage = "rethinkdb@sha256:4452aadba3e99771ff3559735dab16279c5a352359d79f38737c6fdca941c6e5"
```

Comment:
Why the additional two images?  why different format?

Reply:
They are used in that format in existing tests.

Reply:
They are referenced in our pull test and create test. https://github.com/awslabs/soci-snapshotter/blob/main/integration/create_test.go#L59 and https://github.com/awslabs/soci-snapshotter/blob/main/integration/push_test.go#L90